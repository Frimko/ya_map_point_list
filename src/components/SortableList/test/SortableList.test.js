import React from 'react';
import { shallow } from 'enzyme';
import { SortableList } from '../index';

describe('SortableList component', () => {

  let items = [
    {
      id: '6565656543',
      name: 'Это метка1',
    },
    {
      id: '6565656544',
      name: 'Это метка2',
    },
  ];

  describe('snapshot', () => {
    test('snapshot', () => {
      const onDelete = jest.fn().mockName('delete');
      const submit = jest.fn().mockName('submit');
      expect(shallow(<SortableList items={items} submit={submit} deleteMark={onDelete}/>)).toMatchSnapshot();
    });
  });
});
