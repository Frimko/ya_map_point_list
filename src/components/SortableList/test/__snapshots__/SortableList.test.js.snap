// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortableList component snapshot snapshot 1`] = `
<List
  bordered={true}
  dataSource={
    Array [
      Object {
        "id": "6565656543",
        "name": "Это метка1",
      },
      Object {
        "id": "6565656544",
        "name": "Это метка2",
      },
    ]
  }
  header={
    <Form(FormPlacemark)
      submit={[MockFunction submit]}
    />
  }
  loading={false}
  pagination={false}
  prefixCls="ant-list"
  renderItem={[Function]}
  size="small"
  split={true}
/>
`;
